mithril-aggregator/src/database/migration.rs

```diff
@@ -812,5 +812,20 @@ pragma foreign_keys=true;
 alter table certificate drop column immutable_file_number;
         "#,
         ),
+        // Migration 30
+        // Alter `signed_entity` table to add `network` in `artifact` JSON field (for snapshot only).
+        SqlMigration::new(
+            30,
+            r#"
+update signed_entity
+    set artifact = json_insert(
```

Comment:
Good question, since we use serde to deserialize those json values this doesn't matter from a purely functional standpoint as this library don't depend on the order.
There's one case where it can be a little disturbing: on a upgraded server if you do a select where you display the raw json you will see the field at its expected position for newly inserted data but at the end for migrated data.